src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SwaggerGenerator.cs

```diff
@@ -284,7 +284,7 @@ private async Task<Dictionary<OperationType, OpenApiOperation>> GenerateOperatio
             {
                 throw new SwaggerGeneratorException(string.Format(
                     "Conflicting method/path combination \"{0} {1}\" for actions - {2}. " +
-                    "Actions require a unique method/path combination for Swagger/OpenAPI 3.0. Use ConflictingActionsResolver as a workaround",
+                    "Actions require a unique method/path combination for Swagger/OpenAPI 2.0 and 3.0. Use ConflictingActionsResolver as a workaround or write your own PathGroupSelector",
```

Comment:
OpenAPI 2.0 specs are not clear for this like in 3.0:

![image](https://github.com/user-attachments/assets/0c390238-fadd-4866-a214-1ee77089daf4)


But I look back in Swashbuckle code. The same IF statement was there in 2015. OpenAPI 3.0 was released in 2017.

![image](https://github.com/user-attachments/assets/7fbcb360-a0a1-4c80-856a-b64da0f7010a)